packages/core/admin/server/policies/index.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   isAuthenticatedAdmin: require('./isAuthenticatedAdmin'),
   hasPermissions: require('./hasPermissions'),
   isTelemetryEnabled: require('./isTelemetryEnabled'),
+  hasLicenseLimitAllowance: require('./hasLicenseLimitAllowance'),
```

Comment:
Could we rename that into something more specific, like `hasLicenseSeatsLimitAllowance`?